bundles/org.openhab.ui/web/src/components/widgets/widget-mixin.js

```diff
@@ -122,6 +122,7 @@ export default {
             theme: this.$theme,
             themeOptions: this.$f7.data.themeOptions,
             device: this.$device,
+            screen: this.getPageContentRect(),
```

Comment:
New additions to the context should also be added to _hint-components.js_ for the autocompletion dropdown, as well as the docs:
https://www.openhab.org/docs/ui/building-pages.html#dynamically-configuring-components-with-expressions
(just realizing Number is missing there).

Reply:
I added it to `hint-components.js`.

Docs updates are on the way (see https://github.com/florian-h05/openhab-docs/commit/4e611c49c0a679c254f9e8eb6da5366ca2ff5e6c).